src/Microsoft.AspNetCore.Sockets/Transports/WebSocketsTransport.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.AspNetCore.Sockets.Transports
 {
     public class WebSocketsTransport : IHttpTransport
     {
-        public static readonly string Name = "webSockets";
+        public static readonly string TransportName = "webSockets";
```

Comment:
Because so far it's the only one we need to check in a situation where we don't have an instance. See HttpConnectionDispatcher.cs:L27